src/libostree/ostree-repo.c

```diff
@@ -4518,6 +4519,16 @@ find_keyring (OstreeRepo          *self,
       return g_steal_pointer (&file);
     }
 
+  remotes_d_dir = get_remotes_d_dir (self);
+  if (remotes_d_dir)
+    {
+      g_autoptr(GFile) remotes_d = g_file_resolve_relative_path (self->sysroot_dir, remotes_d_dir);
```

Comment:
But here we are prepending sysroot.

I'd say we should do one or the other.  Probably consistently prepend.